imports/api/courses.js

```diff
@@ -30,6 +30,7 @@ const coursePattern = {
   createdAt: Date,
   requireVerified: Match.Maybe(Boolean),
   allowStudentQuestions: Match.Maybe(Boolean),
+  requireApprovedQuestions: Match.Maybe(Boolean),
```

Comment:
Maybe, to be explicit, this (and relevant methods) should be called requireApprovedPublicQuestions